Gems/AtomLyIntegration/EMotionFXAtom/Code/Tools/EMStudio/AnimViewportRenderer.cpp

```diff
@@ -0,0 +1,291 @@
+
+        // Configure tone mapper
+        AzFramework::EntityContextRequestBus::EventResult(
+            m_postProcessEntity, entityContextId, &AzFramework::EntityContextRequestBus::Events::CreateEntity, "postProcessEntity");
+        AZ_Assert(m_postProcessEntity != nullptr, "Failed to create post process entity.");
+
+        m_postProcessEntity->CreateComponent(AZ::Render::PostFxLayerComponentTypeId);
+        m_postProcessEntity->CreateComponent(AZ::Render::ExposureControlComponentTypeId);
+        m_postProcessEntity->CreateComponent(azrtti_typeid<AzFramework::TransformComponent>());
+        m_postProcessEntity->Activate();
+
+        // Init directional light processor
+        m_directionalLightFeatureProcessor = m_scene->GetFeatureProcessor<AZ::Render::DirectionalLightFeatureProcessorInterface>();
+
+        // Init display mapper processor
+        m_displayMapperFeatureProcessor = m_scene->GetFeatureProcessor<AZ::Render::DisplayMapperFeatureProcessorInterface>();
+
+        // Init Skybox
+        m_skyboxFeatureProcessor = m_scene->GetFeatureProcessor<AZ::Render::SkyBoxFeatureProcessorInterface>();
+        m_skyboxFeatureProcessor->Enable(true);
+        m_skyboxFeatureProcessor->SetSkyboxMode(AZ::Render::SkyBoxMode::Cubemap);
+
+        // Create IBL
+        AzFramework::EntityContextRequestBus::EventResult(
+            m_iblEntity, entityContextId, &AzFramework::EntityContextRequestBus::Events::CreateEntity, "IblEntity");
+        AZ_Assert(m_iblEntity != nullptr, "Failed to create ibl entity.");
+
+        m_iblEntity->CreateComponent(AZ::Render::ImageBasedLightComponentTypeId);
+        m_iblEntity->CreateComponent(azrtti_typeid<AzFramework::TransformComponent>());
+        m_iblEntity->Activate();
+
+        // Load light preset
+        AZ::Data::Asset<AZ::RPI::AnyAsset> lightingPresetAsset = AZ::RPI::AssetUtils::LoadAssetByProductPath<AZ::RPI::AnyAsset>(
+            "lightingpresets/default.lightingpreset.azasset", AZ::RPI::AssetUtils::TraceLevel::Warning);
+        const AZ::Render::LightingPreset* preset = lightingPresetAsset->GetDataAs<AZ::Render::LightingPreset>();
+        SetLightingPreset(preset);
+
+        // Create grid
+        AzFramework::EntityContextRequestBus::EventResult(
+            m_gridEntity, entityContextId, &AzFramework::EntityContextRequestBus::Events::CreateEntity, "ViewportGrid");
+        AZ_Assert(m_gridEntity != nullptr, "Failed to create grid entity.");
+
+        AZ::Render::GridComponentConfig gridConfig;
+        gridConfig.m_gridSize = 4.0f;
+        gridConfig.m_axisColor = AZ::Color(0.5f, 0.5f, 0.5f, 1.0f);
+        gridConfig.m_primaryColor = AZ::Color(0.3f, 0.3f, 0.3f, 1.0f);
+        gridConfig.m_secondaryColor = AZ::Color(0.5f, 0.1f, 0.1f, 1.0f);
+        auto gridComponent = m_gridEntity->CreateComponent(AZ::Render::GridComponentTypeId);
+        gridComponent->SetConfiguration(gridConfig);
```

Comment:
Can you get a task up to replace this with a solid ground (using a mesh component and the groundplane_512x512m)? Now that we have this shiny new rendering, we want a solid ground and see a shadow :).